ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -310,4 +318,98 @@ export class IndexerClient {
     };
     return this.queryIndexer(graphqlQuery);
   }
+
+  /**
+   * Queries account's current tokens.
+   * This query queries for any type of tokens (i.e NFT, Fungiable, soul bound, etc).
+   *
+   * @returns GetAccountTokensQuery response type
```

Comment:
Can we provide more details on the resp structure here or link to a doc that does? An example response would be amazing as well

Reply:
it is TypeScript, and `GetAccountTokensQuery` is a type, one can inspect that type (clicking or on hover) and see the response. list the full response with all the fields can be pretty long and spam this file.